awscrt/io.py

```diff
@@ -250,6 +250,14 @@ class TlsVersion(IntEnum):
     DEFAULT = 128  #:
 
 
+class TlsCipherPref(Enum):
+    DEFAULT = (0)
```

Comment:
stuff needs to have SOME kind of docstring or it won't show up in the [API docs](https://awslabs.github.io/aws-crt-python/api/io.html#awscrt.io.TlsVersion)

you need to add a """ triple-quoted docstring, or at the very least a `#:` or the enum value won't even show up

you can see what the docs will look like by locally running `./scripts/make-docs.py`, then look at `docs/index.html` in your browser